src/pages/guest-home-page/GuestHome.jsx

```diff
@@ -31,14 +35,16 @@ const GuestHomePage = () => {
   }, [searchParams, setSearchParams, openModal])
 
   return (
-    <Box data-testid='guestHome' sx={ { flex: 1 } }>
+    <Box data-testid='guestHome' ref={ element } sx={ { flex: 1, overflowY: 'auto' } }>
```

Comment:
You can remove _"data-testid='guestHome'"_ if you don't use it in any test

Reply:
Also, make sure that all the tests are passing 

![image](https://user-images.githubusercontent.com/68649827/220717028-7b646942-e18d-49dd-a570-b75ed67a18bf.png)

Reply:
fixed